content/influxdb/v2.0/process-data/manage-tasks/create-task.md

```diff
@@ -120,3 +121,32 @@ option task = {
 
 # <ctrl-d> to close the pipe and submit the command
 ```
+
+## Create a task using the InfluxDB API
+Provide the following in your API request:
+
+- `/api/v2/tasks` InfluxDB API endpoint
+- **POST** request method
+
+##### Request headers
+- **Content-Type**: application/json
+- **Authorization**: Token _YOURINFLUXDBTOKEN_**
+
+##### Request body
+JSON object with the following fields:
+
+- **flux** : raw Flux task string
+- **orgID**: your [InfluxDB organization ID](/influxdb/v2.0/organizations/view-orgs/#view-your-organization-id)
+- **status**: task status ("active" or "inactive")
+- **description**: task description
+```sh
```

Comment:
I should've left a new line here. This may not render correctly without it.

~~~suggestion
- **description**: task description

```sh
~~~